finished-proposals.md

```diff
@@ -2,16 +2,16 @@
-| [Array.prototype.includes](https://github.com/tc39/Array.prototype.includes/)          | Domenic Denicola, Rick Waldron | [November 2015](https://github.com/rwaldron/tc39-notes/blob/924122cdc03e9ee2afbe8014193f845bddc6da2d/es7/2015-11/nov-17.md#arrayprototypeincludes) | 2016
-| [Exponentiation Operator](https://github.com/rwaldron/exponentiation-operator)         | Rick Waldron                   | [January 2016](https://github.com/rwaldron/tc39-notes/blob/924122cdc03e9ee2afbe8014193f845bddc6da2d/es7/2016-01/2016-01-28.md#5xviii-exponentiation-operator-rw) | 2016
-| [Object.values/Object.entries](https://github.com/tc39/proposal-object-values-entries) | Jordan Harband                 | [March 2016](https://github.com/rwaldron/tc39-notes/blob/80d8837eefdb74ec5532c0fd034c51c83e4b8882/es7/2016-03/march-29.md#objectvalues--objectentries) | 2017
-| [String padding](https://github.com/tc39/proposal-string-pad-start-end)                | Jordan Harband & Rick Waldron  | [May 2016](https://github.com/rwaldron/tc39-notes/blob/master/es7/2016-05/may-25.md#stringprototypepadstartend-jhd) | 2017
-| [Object.getOwnPropertyDescriptors](https://github.com/ljharb/proposal-object-getownpropertydescriptors) | Jordan Harband & Andrea Giammarchi | [May 2016](https://github.com/rwaldron/tc39-notes/blob/master/es7/2016-05/may-25.md#objectgetownpropertydescriptors-jhd) | 2017
-| [Trailing commas in function parameter lists and calls](https://github.com/tc39/proposal-trailing-function-commas)      | Jeff Morrison | [July 2016](https://github.com/rwaldron/tc39-notes/blob/master/es7/2016-07/jul-26.md#9ie-trailing-commas-in-functions) | 2017
-| [Async Functions](https://github.com/tc39/ecmascript-asyncawait)                       | Brian Terlson                  | [July 2016](https://github.com/rwaldron/tc39-notes/blob/master/es7/2016-07/jul-28.md#10iv-async-functions) | 2017
-| [Shared memory and atomics](https://github.com/tc39/ecmascript_sharedmem)              | Lars T Hansen                  | January 2017 | 2017
-| [Lifting Template Literal Restriction](https://github.com/tc39/proposal-template-literal-revision) | Tim Disney         | March 2017 | 2018
+| Proposal                                                                                                           | Champion(s)                        | TC39 meeting notes | Expected Publication Year |
+|--------------------------------------------------------------------------------------------------------------------|------------------------------------|--------------------|---------------------------|
+| [`Array.prototype.includes`](https://github.com/tc39/Array.prototype.includes/)                                    | Domenic Denicola, Rick Waldron     | [November 2015](https://github.com/rwaldron/tc39-notes/blob/master/es7/2015-11/nov-17.md#arrayprototypeincludes) | 2016
```

Comment:
> I actually made the sha links intentional as branch links aren't always as reliable.

They are for this repository, though — these meeting notes aren’t going anywhere after publication, as that would break permalinks.

I simplified the URLs because I noticed some of them were pointing to `master` anyway.

> I think it's very useful to link directly to the commit it was added in.

Useful how? That way you miss out on any follow-up typo/formatting fixes in the notes.